src/main/java/org/mule/tools/devkit/sonar/ConnectorsChecks.java

```diff
@@ -1,21 +1,6 @@
 package org.mule.tools.devkit.sonar;
 
-import org.mule.tools.devkit.sonar.checks.java.BasePackageCheck;
-import org.mule.tools.devkit.sonar.checks.java.DevKitAnnotationsOrderCheck;
-import org.mule.tools.devkit.sonar.checks.java.FunctionalTestPerMetadataCategoryCheck;
-import org.mule.tools.devkit.sonar.checks.java.FunctionalTestPerProcessorCheck;
-import org.mule.tools.devkit.sonar.checks.java.FunctionalTestSuiteCheck;
-import org.mule.tools.devkit.sonar.checks.java.LicenseByCategoryCheck;
-import org.mule.tools.devkit.sonar.checks.java.MetaDataTestCasesExtendAbstractMetaDataTestCaseCheck;
-import org.mule.tools.devkit.sonar.checks.java.NoAssertionsInBeforeOrAfterCheck;
-import org.mule.tools.devkit.sonar.checks.java.NumberOfComplexArgumentsCheck;
-import org.mule.tools.devkit.sonar.checks.java.NumberOfSimpleAndOptionalArgumentsCheck;
-import org.mule.tools.devkit.sonar.checks.java.PayloadDeprecationCheck;
-import org.mule.tools.devkit.sonar.checks.java.RedundantExceptionNameCheck;
-import org.mule.tools.devkit.sonar.checks.java.RefOnlyInComplexTypesCheck;
-import org.mule.tools.devkit.sonar.checks.java.RestCallDeprecationCheck;
-import org.mule.tools.devkit.sonar.checks.java.TestCasesExtendAbstractTestCasesCheck;
-import org.mule.tools.devkit.sonar.checks.java.TestSuiteCheck;
+import org.mule.tools.devkit.sonar.checks.java.*;
```

Comment:
This should be configured in IntelliJ to a different behaviour, not to collapse the imports. There's no way to disable the setting, but it can be configured to only do it when there are more than X, setting X to 500 should do the trick.